circle.yml

```diff
@@ -51,7 +51,8 @@ dependencies:
   override:
     - .circle/docker-compose.sh pull ${DISTRO}
   post:
-    - .circle/docker-compose.sh build ${DISTRO}
+    - .circle/docker-compose.sh build_st2 ${DISTRO}
```

Comment:
Seems it will require changes by @dennybaa, probably entire rake & rspec ruby stuff is affected here:
* https://github.com/StackStorm/st2-packages/blob/master/rake/build/environment.rb
* https://github.com/StackStorm/st2-packages/blob/2c3af10fbbdbee56acf531e369f79fc4e90ee39e/rake/spec/spec_helper.rb


What I know for sure:
- in 1 container they're built in parallel
- in 1 container they're both tested (Rspec smoke tests)